spaceship/lib/spaceship/two_step_or_factor_client.rb

```diff
@@ -25,7 +25,7 @@ def handle_two_step_or_factor(response)
     end
 
     def handle_two_step(response)
-      if response.body.fetch("securityCode", {})["tooManyCodesLock"].to_s.length > 0
```

Comment:
Is `tooManyCodesLock` not a thing anymore? 🤔 Should we maybe keep checking for it just in case?

Reply:
@bolom Just wanted to check in to see your thoughts on this? 😊

Reply:
should we use securityCodeLocked for 4 digit and tooManyCodesSent for 6 digits?